numba_dppy/tests/test_debuginfo.py

```diff
@@ -99,6 +100,7 @@ def foo(var_a, var_b, var_c):
 
     ir_tags = (ir_tag_var_a, ir_tag_var_b, ir_tag_var_c, ir_tag_var_i)
 
+    opt_curr_value = os.environ.get("NUMBA_OPT", 3)  # 3 is a default value
```

Comment:
Do not use both `os.environ` and `config`. They do the same from different levels.